package.json

```diff
@@ -122,7 +124,7 @@
     "mini-css-extract-plugin": "^2.9.2",
     "minisearch": "^3.0.2",
     "mobx": "<6.13.0",
-    "mobx-react": "7.6.0",
+    "mobx-react": "9.2.0",
```

Comment:
Can the version be `^9.2.0`, or does minor revisions of mobx-react tend to break things?